pos_account_wallet_coupon/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 {
     "name": "POS Account Wallet Coupon",
     "version": "14.0.1.0.0",
-    "author": "Acsone SA/NV, Odoo Community Association (OCA)",
+    "author": "ACSONE SA/NV, Odoo Community Association (OCA)",
```

Comment:
At this stage, OCA is not involved in this.
```suggestion
    "author": "ACSONE SA/NV",
```

Reply:
![image](https://user-images.githubusercontent.com/19529533/136531852-ef13db01-c34d-413a-9b44-b68960fbab75.png)

@sbidoul What should be changed to remove that check ?

Reply:
Ok, found

Reply:
https://github.com/acsone/acsone-addons/pull/164/commits/7ec2ec6baa6ad308e406ed6b3aeb9a1eadd9b5fa#diff-28b25500282c29149fccf139833b6435b868cef97f63ecebeeda840ad1ba910aR7